zookeeper-client/zookeeper-client-c/tests/TestOperations.cc

```diff
@@ -126,7 +126,7 @@ class Zookeeper_operations : public CPPUNIT_NS::TestFixture
         zh=zookeeper_init("localhost:2121",watcher,10000,TEST_CLIENT_ID,0,0);
         CPPUNIT_ASSERT(zh!=0);
         // simulate connected state
-        forceConnected(zh);
+        forceConnected(zh, &timeMock.tv);
```

Comment:
this is not related to logback change.  shouldn't this be different CL# which fixes just test issues?

Reply:
With this change, everything is completely tested? or do you have more changes?

Reply:
Nope yes basically CI should be passed.